src/validations/constraints/unit-tests/response-point-FAIL.yaml

```
@@ -0,0 +1,10 @@
+# driver for the response-point unit test
+test-case:
+  name: Test Valid Response Point Use
+  description: Test that the resolved profile contains valid response-points.
+  content: ../content/test-valid-response-point-use.xml # the content to run
```

see other comment on naming convention for validation constraint content file names.  If the file is changes, please also update the reference here

```suggestion
content: ../content/profile-response-point-INVALID.xml # the content to run
```